src/basic/namespace-util.c

```diff
@@ -479,6 +480,24 @@ int namespace_open_by_type(NamespaceType type) {
         return fd;
 }
 
+int namespace_is_init(NamespaceType type) {
+        assert(type >= 0);
+        assert(type <= _NAMESPACE_TYPE_MAX);
+
+        if (namespace_info[type].root_inode == 0)
```

Comment:
I'd treat this as programming error if ever called

Reply:
dunno, it makes the test much easier to write.